content/en/news/releases/1.12.x/announcing-1.12/change-notes/index.md

```diff
@@ -0,0 +1,165 @@
+  ([Issue #33472](https://github.com/istio/istio/issues/33472))
+
+- **Added** Auto mTLS support for workload level peer authentication. You no longer need to configure destination rule when servers are configured with workload level peer authentication policy. This can be disabled by setting `ENABLE_AUTO_MTLS_CHECK_POLICIES` to "false".
+  ([Issue #33809](https://github.com/istio/istio/issues/33809))
+
+- **Added** the support of integrating with GKE workload certificates.
+  ([Issue #35385](https://github.com/istio/istio/issues/35385))
+
+- **Added** values to the Istio Gateway Helm charts for configuring annotations on the ServiceAccount.  Can be used to enable [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) on AWS EKS.
+  ([Pull Request #33914](https://github.com/istio/istio/pull/33914))
+
+- **Added** support for JWT claim based routing on ingress gateways.
+  ([Pull Request #35762](https://github.com/istio/istio/pull/35762))
+
+- **Fixed** the `EnvoyExternalAuthorizationHttpProvider` to match HTTP headers in a case-insensitive way.
+  ([Issue #35220](https://github.com/istio/istio/issues/35220))
+
+## Telemetry
+
+- **Fixed** an issue with `WorkloadGroup` and `WorkloadEntry` labeling of canonical revision.
+  ([Issue #34395](https://github.com/istio/istio/issues/34395))
+
+## Extensibility
+
+- **Added** support for Istio `WasmPlugin` API
+  ([Pull Request #33374](https://github.com/istio/istio/pull/33374))
+
+## Installation
+
+- **Updated** `istioctl tag set default` to control which revision handles Istio resource validation. The revision indicated
+through the default tag will also win leader elections and assume singleton cluster responsibilities.
+  ([Pull Request #35286](https://github.com/istio/istio/pull/35286))
+
+- **Added** labels on pod level for istio-operator and istiod.
+  ([Issue #33879](https://github.com/istio/istio/issues/33879))
+
+- **Added** pilot service annotations on helm chart
+  ([Issue #35229](https://github.com/istio/istio/issues/35229))
+
+- **Added** Support arm64 API for operator, add nodeAffinity arm64 expression.
+  ([Pull Request #35648](https://github.com/istio/istio/pull/35648))
+
+- **Fixed** a bug where specifying same port number with different protocols (TCP and UDP)
+lead to incorrect merging and rendered erroneous manifest.
+  ([Issue #33841](https://github.com/istio/istio/issues/33841))
+
+- **Fixed** Istioctl does not wait on CNI DaemonSet update
+  ([Issue #34811](https://github.com/istio/istio/issues/34811))
+
+- **Fixed** No Permission to list `ServiceExport` from remote clusters in primary cluster.
+  ([Issue #35068](https://github.com/istio/istio/issues/35068))
+
+## istioctl
+
+- **Improved** Analyzers reports output to match the naming scheme expected by the API, i.e `<ns>/<name>` instead of `<name>.<ns>`.
```

Comment:
```suggestion
- **Improved** analyzers report output to match the naming scheme expected by the API, i.e `<ns>/<name>` instead of `<name>.<ns>`.
```